tests/acceptance/user-can-waive-hearings-test.js

```diff
@@ -70,6 +72,8 @@ module('Acceptance | user can waive hearings', function(hooks) {
     assert.ok(find('[data-test-hearings-waived-message]'));
     assert.ok(find('[data-test-button="submitRecommendation"]'));
 
+    assert.equal(this.server.db._collections[2]._records[0].dcpIspublichearingrequired, 'No');
```

Comment:
Let's look into this more, there _should_ be a public getter

Reply:
@trbmcginnis 
<img width="1144" alt="Screen Shot 2019-11-15 at 2 14 29 PM" src="https://user-images.githubusercontent.com/5004319/68969103-34d26c00-07b2-11ea-9cba-25665384d2fa.png">

I'm not sure why you were unable to use the public method for this. I can't recreate this. One think that may have happened is that you were console logging `this.server.db`, and didn't see a concrete property. I think these collections are defined as getters, so you won't see them except at the bottom faded out.

Reply:
I'm going to fix this in an upcoming PR. Thanks!